vignettes/tooltips-popovers/index.Rmd

```diff
@@ -0,0 +1,451 @@
+---
+title: "Tooltips & Popovers"
+resource_files:
+  - examples/tooltips-popovers
```

Comment:
```suggestion
  - examples
```

This should be enough now. I took out the extra nesting by document name now that these examples are stored in the article's subdir.